### ISSUE TITLE
Problems during fine-tuing on my own ATAC-seq data

### ISSUE THREAD

**Issue by Aut-eve** [lucidrains/enformer-pytorch#37]
Hi, I'm now trying to fine-tune the pretrained model on my own ATAC-seq data using HeadAdapterWrapper. Due to device limitations and long training time, I set finetune_last_n_layers_only to 2, and use Adam as optimizer. I use CosineAnnealingLR as learning rate scheduler, and set my learning rate from 1e-3 to 1e-5. However, the result seems not good, so may I have your learning rate, optimizer and other hyper-parameters? Thanks very much.
